src/firmware_new/boards/arduino_uno/echo_library.cc

```diff
@@ -0,0 +1,21 @@
+
+#ifdef PLATFORMIO_BUILD
+#include "echo_library.h"
+#else
+#include "echo_library.h"
```

Comment:
please add comments on this ifdef/include thing like we have in serial_echo_example.cc